csharp/ReplTest/SparkCLRTestEnvironment.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using AdapterTest;
+using AdapterTest.Mocks;
+using Microsoft.Spark.CSharp.Interop;
+using Microsoft.Spark.CSharp.Interop.Ipc;
+using NUnit.Framework;
+
+namespace ReplTest
+{
+    [SetUpFixture]
+    public class SparkCLRTestEnvironment
```

Comment:
@skaarthik 

Test results show that `[SetUpFixture]` did not take effect if it is not defined in the same project.